flink-runtime/src/test/java/org/apache/flink/runtime/highavailability/AbstractHaServicesTest.java

```diff
@@ -42,20 +43,18 @@
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.Executor;
 
-import static org.hamcrest.Matchers.contains;
-import static org.hamcrest.Matchers.is;
-import static org.junit.Assert.assertThat;
-import static org.junit.Assert.fail;
+import static org.assertj.core.api.Assertions.assertThat;
 
 /** Tests for the {@link AbstractHaServices}. */
 public class AbstractHaServicesTest extends TestLogger {
 
     /**
-     * Tests that we first delete all pointers from the HA services before deleting the blobs. See
-     * FLINK-22014 for more details.
+     * Tests that we first delete all pointers from the HA services before deleting the blobs and HA
+     * storage path. See FLINK-22014 for more details.
      */
     @Test
-    public void testCloseAndCleanupAllDataDeletesBlobsAfterCleaningUpHAData() throws Exception {
+    public void testCloseAndCleanupAllDataDeletesBlobsAndHaStoragePathAfterCleaningUpHAData()
```

Comment:
```suggestion
    void testCloseAndCleanupAllDataDeletesBlobsAndHaStoragePathAfterCleaningUpHAData()
```
All test methods should be package-private.